### DISCUSSION TITLE
monitoring nebula K8S resources

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#4640] [Q&A]
Hi @wey-gu 

We want to extend our existing monitoring to include Nebula resources such as node, service and space.
How can we do this, can you please point me to any documentation

**Comment by wey-gu** [comment]
You could deploy the nebulagraph [dashboard](https://github.com/vesoft-inc/nebula-dashboard) or leverage https://github.com/vesoft-inc/nebula-stats-exporter directly.

The dashboard, for now, is designed for non-k8s deployment, but the service/space/nebulagraph metrics should be working with it in k8s env, too.

On the other hand, the stats-exporter makes it possible to hack in any monitor ways we like.
